maven-javadoc-plugin failed to render new @checkstyle tag

```
/home/runner/work/download-maven-plugin/download-maven-plugin/src/main/java/com/googlecode/download/maven/plugin/internal/checksum/Checksums.java:53: error: unknown tag: checkstyle
     * @checkstyle ParameterName (10 lines)
     * 
```

This prevents a release

--- a/checkstyle-ruleset.xml
+++ b/checkstyle-ruleset.xml
@@ -125,11 +125,6 @@
         <property name="optional" value="false"/>
     </module>
     <module name="SuppressWarningsFilter"/>
-    <module name="SuppressWithPlainTextCommentFilter">
-        <property name="offCommentFormat" value="@checkstyle ([\w\|]+) off"/>
-        <property name="onCommentFormat" value="@checkstyle ([\w\|]+) on"/>
-        <property name="checkFormat" value="$1"/>
-    </module>
     <module name="LineLength">
         <property name="fileExtensions" value="java"/>
         <property name="max" value="120"/>
@@ -139,11 +134,6 @@
     <module name="TreeWalker">
         <module name="SuppressWarningsHolder"/>
 
-        <module name="SuppressWithNearbyCommentFilter">
-            <property name="commentFormat" value="@checkstyle (\w+) \((\d+) lines?\)"/>
-            <property name="checkFormat" value="$1"/>
-            <property name="influenceFormat" value="$2"/>
-        </module>
         <!-- Checks for annotations. -->
         <module name="AnnotationUseStyle">
             <property name="elementStyle" value="compact_no_array"/>
@@ -237,7 +227,7 @@
         <module name="MissingJavadocType"/>
         <module name="JavadocStyle"/>
         <module name="AtclauseOrder">
-            <property name="tagOrder" value="@param, @return,                 @throws, @exception, @see, @since, @serial, @serialField,                 @serialData, @deprecated, @todo, @checkstyle"/>
+            <property name="tagOrder" value="@param, @return,                 @throws, @exception, @see, @since, @serial, @serialField,                 @serialData, @deprecated, @todo"/>
         </module>
         <!-- Metrics. -->
         <module name="BooleanExpressionComplexity"/>

--- a/src/main/java/com/googlecode/download/maven/plugin/internal/ArtifactMojo.java
+++ b/src/main/java/com/googlecode/download/maven/plugin/internal/ArtifactMojo.java
@@ -246,8 +246,8 @@ public final class ArtifactMojo extends AbstractMojo {
      * of completion stages, which will contain the downloaded artifacts when complete.
      * @param artifact The artifact for which transitive dependencies need to be downloaded.
      * @return List of completion stages, which will contain the downloaded artifacts when complete.
-     * @checkstyle AnonInnerLength (50 lines)
      */
+    @SuppressWarnings("checkstyle:AnonInnerLength")
     private List<CompletionStage<Artifact>> resolveDependencyArtifacts(final Artifact artifact) {
         final Artifact pomArtifact = this.artifactFactory.createProjectArtifact(
             artifact.getGroupId(),

--- a/src/main/java/com/googlecode/download/maven/plugin/internal/HttpFileRequester.java
+++ b/src/main/java/com/googlecode/download/maven/plugin/internal/HttpFileRequester.java
@@ -94,6 +94,7 @@ public final class HttpFileRequester {
      * Downloads the resource with the given URI to the specified local file system location.
      * @param outputFile The output file.
      * @param headers List of headers.
+     * @throws IOException Thrown if I/O operations don't succeed.
      */
     public void download(final File outputFile, final List<Header> headers) throws IOException {
         try (CloseableHttpClient httpClient = this.createHttpClientBuilder().build()) {

--- a/src/main/java/com/googlecode/download/maven/plugin/internal/SSLProtocols.java
+++ b/src/main/java/com/googlecode/download/maven/plugin/internal/SSLProtocols.java
@@ -56,8 +56,8 @@ public final class SSLProtocols {
     /**
      * Define runtime supports {@code TLS1.3}.
      * @return True if supports.
-     * @checkstyle MethodName (5 lines)
      */
+    @SuppressWarnings("checkstyle:MethodName")
     private static boolean isTls13Supported() {
         try {
             return Arrays.asList(SSLContext.getDefault().getSupportedSSLParameters().getProtocols())

--- a/src/main/java/com/googlecode/download/maven/plugin/internal/checksum/Checksums.java
+++ b/src/main/java/com/googlecode/download/maven/plugin/internal/checksum/Checksums.java
@@ -50,9 +50,8 @@ public final class Checksums {
      * @param sha256 Supplied sha256 checksum, may be {@literal null}.
      * @param sha512 Supplied sha512 checksum, may be {@literal null}.
      * @param log Logger.
-     * @checkstyle ParameterName (10 lines)
      */
-    @SuppressWarnings("checkstyle:LineLength")
+    @SuppressWarnings({"checkstyle:LineLength", "checkstyle:ParameterName"})
     public Checksums(
         @Nullable final String md5, @Nullable final String sha1,
         @Nullable final String sha256, @Nullable final String sha512,
@@ -72,11 +71,11 @@ public final class Checksums {
      * @return True if the file matches all supplied checksums
      *  or if no checksums were supplied.
      */
+    @SuppressWarnings("checkstyle.IllegalCatch")
     public boolean isValid(final File file) {
         boolean valid = true;
         try {
             this.validate(file);
-            // @checkstyle IllegalCatch (1 line)
         } catch (final Exception ex) {
             valid = false;
         }
@@ -105,8 +104,8 @@ public final class Checksums {
      * @param sha512 Supplied sha512 checksum, may be {@literal null}.
      * @return A map of a checksum type to a digest; {@literal null} digests
      *  are not included.
-     * @checkstyle ParameterName (10 lines)
      */
+    @SuppressWarnings("checkstyle:ParameterName")
     private static Map<Checksum, String> create(
         @Nullable final String md5, @Nullable final String sha1,
         @Nullable final String sha256, @Nullable final String sha512

